test/artifact/inspec_detect_test.rb

```diff
@@ -0,0 +1,12 @@
+require "minitest/autorun"
+
+class TestArtifactDetect < Minitest::Test
+  def test_detect
+    out, err = capture_subprocess_io {
+      assert system("inspec detect --no-color")
+    }
+
+    assert_match %r%Platform Details%, out
```

Comment:
won't chefstyle get pissy about using `%r` when it doesn't have `/` in the pattern?

Huh. apparently not. But it does  blow up on the use of curlies on the assertion above. I agree with the curlies. so you might want to disable that check. I think that's the only blocker to getting the tests running.